investos/portfolio/risk_model/factor_risk.py

```diff
@@ -1,23 +1,184 @@
 import cvxpy as cvx
 import numpy as np
-
+import pandas as pd
+import numpy as np
+import statsmodels.api as sm
 import investos.util as util
+import time
+import os
+import forecastos as fos # from pypi
+from functools import reduce
+from scipy.stats import mstats
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LinearRegression
 from investos.portfolio.risk_model import BaseRisk
+from investos.portfolio.risk_model.factor_utils import *
+from investos.portfolio.risk_model.factor_covariance_adjustment import *
+from investos.portfolio.risk_model.asset_diagonal_variance_adjustment import *
 
-
+# will comment out this later
 class FactorRisk(BaseRisk):
     """Multi-factor risk model."""
 
     def __init__(
-        self, factor_covariance, factor_loadings, idiosyncratic_variance, **kwargs
-    ):
+        self, start_date,end_date, risk_model_window: int,  recalc_freq: int, fos_risk_factor_uuids: list = [], fos_return_uuid: list = ["return_1d"],
+        adjustments: dict = {'FactorCovESTArgs':{'window':21, 'half_life': 360}},
+        factor_covariance = None, factor_loadings = None, idiosyncratic_variance = None,
+        **kwargs):
         super().__init__(**kwargs)
+        self._risk_model_window = risk_model_window             #pd.Timedelta(days = risk_model_window)
+        self._recalc_freq = recalc_freq                         #pd.Timedelta(days = recalc_freq)
+
+        self._fos_risk_factor_uuids = fos_risk_factor_uuids
+        self._fos_return_uuid = fos_return_uuid
+
+        self._end_date = end_date
+        self._start_date = start_date
+        self._adjustments = adjustments
+        #self._drop_excluded_assets()
+
+        if self._fos_risk_factor_uuids: 
+            print("=================Barra Risk Model================= ")
+            print("\n1. Initialization")
```

Comment:
Print statements should be more descriptive of what is going on.

Reply:
resolved